Sources/SwiftSyntax/Syntax.swift

```diff
@@ -454,6 +488,40 @@ public extension SyntaxProtocol {
     return raw.contentLength
   }
 
+  /// The byte source range of this node including leading and trailing trivia.
+  var totalByteRange: ByteSourceRange {
+    return ByteSourceRange(offset: position.utf8Offset, length: totalLength.utf8Length)
+  }
+
+  /// The byte source range of this node excluding leading and trailing trivia.
+  var contentByteRange: ByteSourceRange {
+    return ByteSourceRange(
+      offset: positionAfterSkippingLeadingTrivia.utf8Offset,
+      length: contentLength.utf8Length
+    )
+  }
+
+  @available(*, deprecated, renamed: "totalByteRange")
+  var byteRange: ByteSourceRange {
+    return ByteSourceRange(offset: position.utf8Offset, length: totalLength.utf8Length)
+  }
+
+  /// The textual byte length of this node including leading and trailing trivia.
+  @available(*, deprecated, message: "Use totalLength.utf8Length")
+  var byteSize: Int {
```

Comment:
FWIW I've always found `SourceLength` and `utf8Length` very confusing. It's really `numUTF8CodeUnits` or something similar. For some reason `utf8Length` makes my brain think "length of the string when the given bytes are decoded as utf8". Maybe it's just the `length` that does that to me. If no one else has this problem I'm happy to drop it. I figured I'd bring it up since you're renaming here anyway.